### PR TITLE
Morph UI search + food digesting

### PR DESCRIPTION
**Pull request by Absolucy**
## About The Pull Request

This adds a search bar to the morph stomach UI, and allows morphs to digest food items for small amounts of healing. Said healing is rather small, and is _capped_ - a morph can only heal 20% of its max health over the course of 2.5 minutes using food (this can be nerfed further if needed).

Also, morph digestion victims can now be properly fixed with synthflesh or rezadone.

## Why It's Good For The Game

Searching through items is _very_ useful when you've eaten like 500 things.

Food digesting gives morphs a way to patch minor wounds without having to waste a monkey or station pet on it.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

**[VIDEO HAS OLD SEARCH BAR]**
https://github.com/BeeStation/BeeStation-Hornet/assets/65794972/77650c7a-211e-42cf-894d-c667f7c49842

</details>

## Changelog
:cl:
add: Added a search bar to the morph stomach UI.
add: Morphs can now digest nutritious food to recover small amounts of health, however this healing is capped - a morph can only recover up to 20% of its health over the course of 2.5 minutes by digesting food.
tweak: Being husked by morph digestion is now considered a burn husk, meaning that synthflesh or rezadone can fix morph digestion victims.
/:cl:

### CODE REVIEW

**Review by itsmeow — APPROVED** [review]

**Comment by itsmeow** [comment]
Needs a search icon to make it more obvious
also wider search bar, it's super tiny

**Review by itsmeow — CHANGES_REQUESTED** [review]
above

**Comment by Absolucy** [comment]
@itsmeow 
image

File: tgui/packages/tgui/interfaces/Morph.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
+import { createSearch } from 'common/string';
 import { useBackend, useLocalState } from '../backend';
 import { Window } from '../layouts';
-import { Box, Button, Section, Tabs, LabeledList } from '../components';
+import { Input, Button, Section, Tabs, LabeledList, Box, Icon } from '../components';
 
-export const Morph = (props, context) => {
+export const Morph = (_props, _context) => {
```
Comment (itsmeow) [inline tgui/packages/tgui/interfaces/Morph.js]: these are useless, why add them

Comment (Absolucy) [reply tgui/packages/tgui/interfaces/Morph.js]: force of habit

**Review by itsmeow — APPROVED** [review]

**Pull request merged**